snippets/mark-fields-as-touched.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Mark reactive form fields as touched to highlight them
+
+tags:
+  - reactive forms
+  - reactive forms validation
+  - tips
+---
+
+# Content
+Here is the way to notify user that there are fields with non-valid values.
+
+**markFieldsAsTouched** function takes _control_ which is FormGroup type as arg. 
+
+```typescript
+@Component({
+  selector: 'app-root',
+  template: `
+      <form [formGroup]="yourAwesomeComponentReactiveForm">
+          ...
+          <button (click)="markFieldsAsTouched(yourAwesomeComponentReactiveForm)"> Submit </button>
+      </form>
+   `,
+})
+export class AppComponent {
+  yourAwesomeComponentReactiveForm: FormGroup;
+  
+  markFieldsAsTouched(formGroup: FormGroup | FormArray): void {
```

Comment:
I'd move this out as a reuseable function, this way you don't have to have all the component stuff in the #content, only in the interactive example